### PR TITLE
45 parse supports getrelated

### PR DESCRIPTION
**Pull request by Libisch**
Add `ClearmashRelatedDocuments` to enable `parse_clearmash_document` to support RelatedDocuments type fields:
* related_documents object
* get related documents by field
* get IDs from first page of results

### CODE REVIEW

File: tests/mocks/clearmash-api-documents-get-238721.json [hunk 1]
```diff
@@ -0,0 +1,5 @@
+{
```
Comment (Libisch) [inline tests/mocks/clearmash-api-documents-get-238721.json]: (Should have deleted this file before PR)

File: tests/test_clearmash_api.py [hunk 2]
```diff
@@ -161,3 +161,48 @@ def test_get_related_docs_of_item():
...
+    '9eecbf049e5e431cbab8445d5e429dc7',
+    'f3724ce76a6d47dc8613e3dfe4f9b1dd',
+    '4026c28c04e94373b02410b95169c3fc']
+    all_related_by_field = related_documents.get_related_documents(220590, "_c6_beit_hatfutsot_bh_base_template_multimedia_photos")
+    assert len(all_related_by_field["Entities"]) == 13
+    first_doc = all_related_by_field["Entities"][1]
+    assert first_doc["Metadata"]["Id"] == 136844
+
+
+
```
Comment (Libisch) [inline tests/test_clearmash_api.py]: (commented dirt will be removed)

**Comment by Libisch** [comment]
@OriHoch I'm not done yet, just wanted to see if this is the right direction (though all test passed locally)

**Comment by OriHoch** [comment]
Looks good, it's the right direction

Only comment is that the code can be much simpler and focused

**Comment by Libisch** [comment]
@OriHoch

**Comment by Libisch** [comment]
Replaced by #57

**Pull request closed without merging**